### ISSUE TITLE
Hide codename on submission page, reveal with hover

### ISSUE THREAD

**Issue by Hainish** [freedomofpress/securedrop#380]
On the submission page, we have the following text:

```
Remember! Your code name is: vivo icky dyne 13th tung dyne nil amuse
```

We probably want to hide this code name and only reveal it if the source wants it shown.  I would see this as desirable if the source is in a public place while submitting documents, a likely scenario.  Perhaps we should hide the code name unless the source hovers over it?

**Comment by garrettr** [comment]
This wasn't merged for 0.3, but I think we should merge it for 0.4 and let it bake on develop for a while to get feedback.

**Comment by redshiftzero** [comment]
It looks like this was implemented in #1215. Closing.
